django_ses/management/commands/get_ses_statistics.py

```diff
@@ -32,6 +32,8 @@ def handle(self, *args, **options):
         connection = SESConnection(
             aws_access_key_id=settings.ACCESS_KEY,
             aws_secret_access_key=settings.SECRET_KEY,
+            proxy=settings.AWS_SES_PROXY,
```

Comment:
I've added the tests as you proposed. I tried simply storing the args to `__init__()` to the object, but somehow couldn't read them back later on. Appending them to the outbox works though, so I used that.